### ISSUE TITLE
Misleading message about 7z command

### ISSUE THREAD

**Issue by enumag** [composer/composer#9994]
I got this message from composer:

```
As there is no 'unzip' nor '7z' command installed zip files are being unpacked using the PHP zip extension.
This may cause invalid reports of corrupted archives. Besides, any UNIX permissions (e.g. executable) defined in the archives will be lost.
Installing 'unzip' or '7z' may remediate them.
```

The `7z` command however does exist in the system - I ran `apt-get install p7zip-full` to get it.

After checking composer's source code it seems that 7z is only [checked](https://github.com/composer/composer/blob/c65890d2d3364c62236651bb277987e00835b608/src/Composer/Downloader/ZipDownloader.php#L43-L45) for on Windows but ignored on linux. So on linux I actually have to install unzip.

The message is highly misleading this way. Either it shouldn't mention 7z or 7z should be accepted on any platform.

**Comment by enumag** [comment]
@Seldaek I suspect this closes #9951 as well, right?

**Comment by Seldaek** [comment]
Hmm right, I thought there was something already..  I guess we need to look up 7zz too then.

**Comment by MightyPork** [comment]
Is this really fixed? I just saw this warning in composer 2.3.1 with both unzip and 7z installed on Arch Linux, in the standard path, `/usr/bin`. Where is composer trying to find them?

![Screenshot_20220426_174417](https://user-images.githubusercontent.com/2041118/165340833-75a7c265-bf45-46f8-a39f-67545ad5d2c2.png)

![Screenshot_20220426_174442](https://user-images.githubusercontent.com/2041118/165340857-71c81103-b77e-4abe-8f74-06157f5b36eb.png)

**Comment by Seldaek** [comment]
@MightyPork try with latest Composer perhaps, but also check your PATH is correct when running php/composer? It essentially looks up in the PATH.
